datamations/datamation_groupby.py

```diff
@@ -466,73 +505,100 @@ def prep_specs_summarize(self, width=300, height=300):
+                                if self.operations[-1] == "mean":
+                                    min_array.append(self._output[y_axis][group][subgroup][l3group] - self._error[y_axis][group][subgroup][l3group])
+                                else:
+                                    min_array.append(self._output[y_axis][group][subgroup][l3group])
+                            if subgroup in self._output[y_axis][group] and l3group in self._output[y_axis][group][subgroup]:
+                                if self.operations[-1] == "mean":
+                                    max_array.append(self._output[y_axis][group][subgroup][l3group] + self._error[y_axis][group][subgroup][l3group])
+                                else:
+                                    max_array.append(self._output[y_axis][group][subgroup][l3group])
+                domain = [round(min(min_array), 13), round(max(max_array), 13)]
+            elif len(self._by) > 1:
+                domain = [
+                    round(min(self._output[y_axis][groups[0]][subgroups[0]] - self._error[y_axis][groups[0]][subgroups[0]], self._output[y_axis][groups[1]][subgroups[1]] - self._error[y_axis][groups[0]][subgroups[0]]),13),
+                    round(max(self._output[y_axis][groups[1]][subgroups[0]] + self._error[y_axis][groups[1]][subgroups[0]], self._output[y_axis][groups[1]][subgroups[1]] + self._error[y_axis][groups[1]][subgroups[1]]),13),
+                ]
+            else:
+                domain = [
+                    round(min(self._output[y_axis][groups[0]]-self._error[y_axis][groups[0]], self._output[y_axis][groups[1]]-self._error[y_axis][groups[1]]),13),
+                    round(max(self._output[y_axis][groups[0]]+self._error[y_axis][groups[0]], self._output[y_axis][groups[1]]+self._error[y_axis][groups[1]]),13)
+                ]
 
-        y_encoding = {
-            "field": "datamations_y",
-            "type": "quantitative",
-            "title": [self.operations[-1] + " of", y_axis] if self.operations[-1] == "median" else self.operations[-1] + "(" + y_axis + ")",
-            "scale": {
-            "domain": domain
+            y_encoding = {
+                "field": "datamations_y",
+                "type": "quantitative",
+                "title": [self.operations[-1] + " of", y_axis] if self.operations[-1] == "median" else self.operations[-1] + "(" + y_axis + ")",
+                "scale": {
+                "domain": domain
+                }
             }
-        }
 
         meta = {
             "axes": len(self._by) > 1,
             "description": "Plot " + self.operations[-1] + " " + y_axis + " of each group" + (", with errorbar" if self.operations[-1] == 'mean' else "") + ", zoomed in"
         }
+        if any(element in ['count', 'quantile'] for element in self.operations):
+            left = self._output[y_axis][groups[0]].max()
```

Comment:
This is problematic since there could be more than grow groups. Can you make it generic hardcoding the index in the list of groups?